config/rbac/role_binding.yaml

```diff
@@ -9,11 +9,11 @@ metadata:
     app.kubernetes.io/part-of: primaza
     app.kubernetes.io/managed-by: kustomize
   name: manager-rolebinding
+  namespace: system
```

Comment:
Where does system come from is this supposed to be a variable with the namespace for this primaza tenant?

Reply:
That's just a placeholder that can be used in kustomize manifests to identify objects. Kustomize's default seemed to be `system` so we are relying on this value. We can choose a different value if we prefer.

Default usage of these manifests would rely on overwriting this value.